netlify.toml

```
@@ -100,15 +100,57 @@
   status = 301
 # ========== END TERMS REDIRECTS ==========
 
-# ========== INDIVIDUAL REDIRECTS ==========
+# ========== BLOG REDIRECTS ==========
 [[redirects]]
-  from = "/contact/sales"
-  to = "/demo"
+  from = "/blog/better-onboarding-advice-from-engineering-leaders/"
+  to = "/blog/better-onboarding-how-to-prevent-codebase-overwhelm/"
   status = 301
 
 [[redirects]]
-  from = "/blog/better-onboarding-advice-from-engineering-leaders/"
-  to = "/blog/better-onboarding-how-to-prevent-codebase-overwhelm/"
+  from = "/go/:post-name"
+  to = "/blog/go/:post-name"
```

Can we use the `:slug` placeholder name?

```suggestion
  from = "/go/:slug"
  to = "/blog/go/:slug"
```